src/components/InputSet/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Input } from '../../variables';
+import { User } from '../../Shared';
+import styles from './styles.module.css';
+
+interface InputsStatesSignature extends User {
+  [key: string]: string;
+}
+
+interface InputSetProps {
+  item: Input;
+  value: string;
+  setValue: React.Dispatch<React.SetStateAction<InputsStatesSignature>>;
+  setErrors: React.Dispatch<React.SetStateAction<string[]>>;
+  errors: string[];
+}
+
+const InputSet: React.FC<InputSetProps> = ({
+  item,
+  value,
+  setValue,
+  setErrors,
+  errors,
+}) => {
+  const {
+    type, name, label, labelId, isValidated, regExp,
+  } = item;
+  const validateInput = (
+    expressionToMath: {
+      [Symbol.match](string: string): RegExpMatchArray | null;
+    },
+    event: React.FocusEvent<HTMLInputElement>,
+  ): boolean => (!!(event.target.value && event.target.value.match(expressionToMath)));
```

Comment:
Please do not use `!!`. It would be better to convert expression to boolean value explicitly.

Reply:
fixed, thanks
But why? :)

Reply:
To improve code readability it would be better to use explicit conversions